Sources/SpeziViews/Views/Button/DismissButton.swift

```diff
@@ -30,29 +31,43 @@ public struct DismissButton: View {
                 dismiss()
             }
         } else {
-            Button {
-                dismiss()
-            } label: {
-                Image(systemName: "xmark")
-                    .font(.system(size: 10, weight: .bold, design: .rounded))
-                    #if !os(watchOS)
-                    .foregroundStyle(.secondary)
-                    #endif
-                    .background {
-                        Circle()
-                            #if os(iOS)
-                            .fill(Color(uiColor: .secondarySystemBackground))
-                            #elseif os(watchOS)
-                            .fill(Color(uiColor: .darkGray))
-                            #endif
-                            .frame(width: 25, height: 25)
-                    }
-                    .frame(width: 27, height: 27) // make the tap-able button region slightly larger
-            }
-            .accessibilityLabel("Dismiss")
-            .buttonStyle(.plain)
+            fallbackButton
+        }
+#endif
+    #else // swift < 6.2
```

Comment:
```suggestion
    #endif
#else // swift < 6.2
```

Reply:
fixed; actually needed to swap the indent; thanks for catching this